code/datums/objective/individual_objective/moebius.dm

```diff
@@ -0,0 +1,190 @@
+	..()
+	units_requested = rand(2,3)
+	desc = "Perform [units_requested] autopsies."
+	RegisterSignal(mind_holder, COMSING_AUTOPSY, .proc/task_completed)
+
+/datum/individual_objective/autopsy/task_completed(mob/living/carbon/human/H) 
+	if(H in cadavers)
+		return
+	cadavers += H
+	..(1)
+
+/datum/individual_objective/autopsy/completed()
+	if(completed) return
+	UnregisterSignal(mind_holder, COMSING_AUTOPSY)
+	..()
+
+/datum/individual_objective/more_research
+	name = "Mandate of Science"
+	req_department = list(DEPARTMENT_SCIENCE, DEPARTMENT_MEDICAL)
+	limited_antag = TRUE
+	rarity = 4
+	var/obj/item/target
+
+/datum/individual_objective/more_research/can_assign(mob/living/L)
+	if(!..())
+		return FALSE
+	return pick_faction_item(L)
+
+/datum/individual_objective/more_research/assign()
+	..()
+	target = pick_faction_item(mind_holder)
+	desc = "\The [target] is wasted in their hands. Put it into a destructive analyzer."
+	RegisterSignal(mind_holder, COMSING_DESTRUCTIVE_ANALIZER, .proc/task_completed)
+
+/datum/individual_objective/more_research/task_completed(obj/item/I) 
+	if(target.type == I.type)
+		..(1)
+
+/datum/individual_objective/more_research/completed()
+	if(completed) return
+	UnregisterSignal(mind_holder, COMSING_DESTRUCTIVE_ANALIZER)
+	..()
+
+/datum/individual_objective/damage
+	name = "A Different Perspective"
+	req_department = list(DEPARTMENT_SCIENCE, DEPARTMENT_MEDICAL)
+	var/last_health
+
+/datum/individual_objective/damage/assign()
+	..()
+	units_requested = rand(120,160)
+	desc = "Receive cumulative [units_requested] damage of any kind, to ensure that you see things in a different light"
+	last_health = mind_holder.health
+	RegisterSignal(mind_holder, COMSIG_HUMAN_HEALTH, .proc/task_completed)
+
+/datum/individual_objective/damage/task_completed(health)
+	if(last_health > health)
+		units_completed += last_health - health
+	last_health = health
+	if(check_for_completion())
+		completed()
+
+/datum/individual_objective/damage/completed()
+	if(completed) return
+	UnregisterSignal(mind_holder, COMSIG_HUMAN_HEALTH)
+	..()
+
+/datum/individual_objective/for_science
+	name = "Call of Science"
+	req_department = list(DEPARTMENT_SCIENCE, DEPARTMENT_MEDICAL)
+	limited_antag = TRUE
+	rarity = 4
+	var/mob/living/carbon/human/target
+	var/list/valid_organs = list()
+
+/datum/individual_objective/for_science/can_assign(mob/living/L)
+	if(!..())
+		return FALSE
+	var/list/candidates = (GLOB.player_list & GLOB.living_mob_list & GLOB.human_mob_list) - L
+	return candidates.len
+
+/datum/individual_objective/for_science/assign()
+	..()
+	var/list/valid_targets = (GLOB.player_list & GLOB.living_mob_list & GLOB.human_mob_list) - mind_holder
+	target = pick(valid_targets)
+	for(var/obj/item/organ/external/E in target.organs)
+		valid_organs += E
+	for(var/obj/item/organ/O in target.internal_organs)
+		valid_organs += O
+	desc = "[target] have something interesting to them. Put any organ of [target] in destructive analyzer"
```

Comment:
dot at the end

Reply:
Also bad wording. Maybe...
"[target] looks interesting. Put any of their organs in destructive analyzer."
also make sure you don't get to chop off their bones or head or heart or groin.